src/App.js

```diff
@@ -11,21 +12,23 @@ class App extends React.Component {
       resources: [],
       isFetched: false,
       isLoading: false,
-      search: '',
-      error: null
+      search: "",
+      error: null,
+      opened: false,
```

Comment:
It seems like `opened` here is a flag that tells you whether the user has explicitly toggled the sort order, or if it should still be the default. That's maybe not a good name, because it took me a while to figure that out. Perhaps it would be easier to simply store which sort order is currently active, starting with the default `'likes'` (or popularity or whatever) and just make that the `else` case when you decide how to display the list?